Gemfile.lock

```diff
@@ -359,6 +359,9 @@ GEM
     rspec-expectations (3.8.2)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.8.0)
+    rspec-its (1.2.0)
```

Comment:
Whats the advantages of using this gem over writing example like `it { is_expected.to have_attributes(status: ok) }`?

Reply:
Not a huge difference, but they read a bit better to me. 
It allows you to do things like:

```ruby
its("phone_numbers.size") { should_not eq(0) }
```

or, it also provide handy methods to pass parameters in a more advanced way than `have_attributes`. [Have a look at the docs](https://github.com/rspec/rspec-its). I personally like it.

I decided to add it, as it is a gem maintain by RSpec team.

Reply:
Sounds good!